lightning/src/ln/reorg_tests.rs

```diff
@@ -427,3 +431,112 @@ fn test_set_outpoints_partial_claiming() {
 		node_txn.clear();
 	}
 }
+
+fn do_test_to_remote_after_local_detection(style: ConnectStyle) {
+	// In previous code, detection of commitment transaction to_remote outputs in a remote
```

Comment:
A bit of repetition, just say "detection of `to_remote` output in remote/counterparty commitment transaction was...". `to_remote` output is a specified output type where "remote" assign a commitment direction, there is only 1 `to_remote` output at best per-commitment ?